toqito/state_metrics/bures_distance.py

```diff
@@ -0,0 +1,68 @@
+"""Bures distance metric."""
+import numpy as np
+
+from toqito.state_metrics import fidelity
+
```

Comment:
There should be an extra blank line between `from toqito.state_metrics import fidelity` and `def bures_distance(rho_1: np.ndarray, rho_2: np.ndarray, decimals: int = 10) -> float:` (PEP-8 requirement).